spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -237,13 +241,12 @@ ArmeriaServerConfigurator actuatorServerConfigurator(
                             new EndpointLinksResolver(endpoints).resolveLinks(req.path());
                     return HttpResponse.ofJson(ACTUATOR_MEDIA_TYPE, ImmutableMap.of("_links", links));
                 };
-                if (managementPort != null) {
-                    sb.virtualHost(managementPort)
-                      .route().addRoute(route).defaultServiceName("LinksService").build(linksService);
-                } else {
+                if (exposedPorts.isEmpty()) {
                     sb.route().addRoute(route).defaultServiceName("LinksService").build(linksService);
+                } else {
+                    exposedPorts.forEach(port -> sb.virtualHost(port).route().addRoute(route)
+                                                   .defaultServiceName("LinksService").build(linksService));
```

Comment:
nit; what do you think of also adding an assertion to verify that `LinksService` is also being served at the internal port in the newly added integration tests?

Reply:
`LinksService` is for exposing `/actuator` end point. It's already validated in the test cases.
https://github.com/line/armeria/pull/3946/files#diff-48bc8efa7c13f57a48da4207ba3a2a0bee144c3c4232a96c0c8279bd8dd00fddR200

So we don't need to do additional validation of just add such a comment into test case to understand easier ? 🤔

Reply:
Ah sorry! Nevermind this comment 😅